### PR TITLE
DOC-10389-Update-SQLPP-Scopes-Collections

### PR DESCRIPTION
**Pull request by dinaabennett**
Updated examples as per Pasin's request in https://issues.couchbase.com/browse/CBL-4054.

### CODE REVIEW

File: modules/ROOT/pages/_partials/commons/common-document.adoc [hunk 1]
```diff
@@ -468,6 +472,7 @@ include::{root-partials}block_tabbed_code_example.adoc[]
 :param-tags!:
 :param-leader!:
 
+You can set expiration for a whole Collection
```
Comment (pasin) [inline modules/ROOT/pages/_partials/commons/common-document.adoc]: We do not support his feature in 3.1. The expiration can be only set to a document in 3.1.

File: modules/ROOT/pages/_partials/commons/common-query-n1ql-mobile.adoc [hunk 2]
```diff
@@ -140,10 +140,10 @@ This behavior is inline with that of Couchbase Server {sqlpp_cbm} -- see example
 | SELECT * FROM _default
 | _default
 
-|SELECT * FROM db
-|db
+|SELECT * FROM datasource
```
Comment (pasin) [inline modules/ROOT/pages/_partials/commons/common-query-n1ql-mobile.adoc]: I couldn't find where we describe what the datasource could be. In general, the datasource code be the following:

* < database-name > : default collection
* _ (underscore)       : default collection
* < scope-name >.< collection-name > : a collection in a scope
* < collection-name > : a collection in the default scope

I have also provided some example in one of the comments in https://issues.couchbase.com/browse/CBL-4054.

File: modules/ROOT/pages/_partials/commons/common-sgw-replication.adoc [hunk 3]
```diff
@@ -30,9 +30,31 @@ Your application runs a replicator (also referred to here as a client), which  w
...
+Each scope can contain one or more collections.
+
+To learn about Scopes and Collections, see xref:{param-module}:database.adoc[]
+
+You can set up a replication scheme across any of the three data levels:
+
+Database:: All scopes and collections in the database are synced,
+including the `_default` scope and the `_default` collection.
+
+Scope:: All the collections in a named scope.
```
Comment (pasin) [inline modules/ROOT/pages/_partials/commons/common-sgw-replication.adoc]: Not sure what this means in this context. However, CBL doesn't provide an API to specify only a scope for syncing all collections in the scope. CBL only provides API to specify a list of collections.

File: modules/ROOT/pages/_partials/commons/common-document.adoc [hunk 4]
```diff
@@ -66,7 +67,7 @@ Always use explicit creation of the expected type, whenever the type of result i
 
 [source, Java]
 ----
-Map<String, Object> doc = db.getDocument(someDoc.getId()).toMap();
+Map<String, Object> doc = collection.getDocument(someDoc.getId()).toMap();
```
Comment (pasin) [inline modules/ROOT/pages/_partials/commons/common-document.adoc]: Only Java?

**Review by franrob-projects — APPROVED** [review]

**Review by osfameron — APPROVED** [review]
Looks good!
Minor nitpicks/question:
Is the extra space inside the `< database-name >` holders part of our style guide?
Ideally put the datasources in code markers (e.g. backticks).

**Pull request merged**